src/Agent/NewRelic/Agent/Core/Config/Configuration.cs

```diff
@@ -1,6 +1,3 @@
-// Copyright 2020 New Relic, Inc. All rights reserved.
```

Comment:
Do we need to restore this copyright header?

Reply:
🤷‍♂️ 
I think because there's a "generated by a tool" comment below, the license isn't required. I definitely don't get any build warnings or errors without it, even though VS is configured to bark if the header isn't present.

Reply:
It would be nice to just use the generated output from the xsd2code tool without manual modifications.

Reply:
I put it back manually for now, it's not too hard to do. But I agree it would be nice if it didn't matter.